evaluate_tflite.py

```diff
@@ -36,7 +36,8 @@
 
 def tflite_worker(model, scorer, queue_in, queue_out, gpu_mask):
     os.environ['CUDA_VISIBLE_DEVICES'] = str(gpu_mask)
-    ds = Model(model, BEAM_WIDTH)
+    ds = Model(model)
+    ds.setBeamWidth(BEAM_WIDTH)
```

Comment:
So we keep forcing hard-coded values (potentially) different from what the model has ? I don't feel safe with that :/

Reply:
@lissyx Why?

Reply:
We force-overwrite default model's values by some random magic constant. That was fine when we had no default value recorded with the model, but now it's just asking for people to make mistake and the complain :)